pandas/core/resample.py

```diff
@@ -86,6 +86,8 @@
     Tick,
 )
 
+from pandas.core.reshape.concat import concat
```

Comment:
your imports are in the wrong order and causing failed checks.

run `> isort pandas`